app/src/main/res/layout/mission_item.xml

```diff
@@ -37,11 +37,8 @@
 				android:layout_alignParentRight="true"
 				android:layout_centerVertical="true"
 				android:layout_marginRight="1dp"
-
 				android:src="?options"
 				android:scaleType="centerInside"
-
-				android:src="@drawable/ic_menu_more"
 				android:scaleType="center"
```

Comment:
I'd like to include this PR into 0.15.0. Please try to run your changes. It looks like there are more unresolved merge conflicts. The build still fails for  `/home/travis/build/TeamNewPipe/NewPipe/app/src/main/res/layout/mission_item.xml:42: error: duplicate attribute`. This time this is about the `scaleType`

Reply:
@TobiGr Okay,I'll try fix it